custom/icds_reports/views.py

```diff
@@ -1658,18 +1656,7 @@ def post(self, request, *args, **kwargs):
 @method_decorator([login_and_domain_required], name='dispatch')
 class InactiveAWW(View):
     def get(self, request, *args, **kwargs):
-        export_file = BytesIO()
-        excel_data = AggregateInactiveAWW.objects.all()
-        columns = [x.name for x in AggregateInactiveAWW._meta.fields] + [
-            'days_since_start',
-            'days_inactive'
-        ]
-        rows = [columns]
-        for data in excel_data:
-            rows.append(
-                [getattr(data, field) or 'N/A' for field in columns]
-            )
-
-        export_from_tables([['inactive AWWSs', rows]], export_file, 'csv')
-        title = "inactive_awws_%s" % date.today().strftime('%Y-%m-%d')
-        return export_response(export_file, 'csv', title)
+        collect_inactive_awws()
+        last_sync = IcdsFile.objects.filter(data_type='inactive_awws').order_by('-file_added').first()
+        zip_name = 'inactive_awws_%s' % datetime.today().strftime('%Y-%m-%d')
+        return export_response(last_sync.get_file_from_blobdb(), 'csv', zip_name)
```

Comment:
It seems possible that the blob will not exist. Do you need to handle `corehq.blobs.exceptions.NotFound` here?